docs/site/LOCAL_DEV.md

```diff
@@ -1,61 +1,66 @@
-# Running a site with the documentation locally
+# Running a site with the documentation locally.
 
-- Don't forget to clone repo firstly.
+- Clone repo firstly.
 
 - Free 80 port to bind.
 
-- Install werf
+- Install werf.
 
-- Open console and start documentation container with one of the following methods:
-  - Using makefile:
+- Open console and running documentation and site containers with one of the following methods:
 
+  - The first method — open two consoles and run containers using the Makefile:
+
+    1. In the first console run:
+		
     ```shell
     cd docs/documentation
     make up 
     ```
```

Comment:
@mortelumina98 Please add indent with three spaces to the code block here and in the similar cases below